index.html

```diff
@@ -11,169 +11,223 @@
     
   </head>
   <body>
-    <div class="maincont">
-
+    <div class="outer-maincont">
       <header class="app-bar">
-        <nav class="lower-bar">
-          <a href="#" class="logoo">Somarib Emmanuel</a> 
-          <div class="menu-icons"><a href="#"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></div>
-        </nav>
+
+          <nav class="lower-bar">
+              <ul><li><a href="#" class="logoo">Somarib Emmanuel</a></li></ul>
+              <ul class="big-menu">
+                  <li><a href="">Portfolio</a></li>
+                  <li><a href="">About</a></li>
+                  <li><a href="">Contact</a></li>
+                  <li><a href=""><img class="" src="image/bg-envlope.png" alt="menu icon"></a></li>
+              </ul>
+              <ul class="menu-icons"><li><a href="#"><img class="menuimg" src="image/menu.png" alt="menu icon"></a></li></ul>
+          </nav>
       </header>
+      <main class="main-cont">
+          <article class="header-section">
+              <div class="header-inner-cont">
+                  <section class="inner-section">
+                      <h1 class="myname">Somarib Emmanuel</h1>
+                      <h2 class="jobtitle">Software Developer</h2>
+                      <p class="abilities">
+                          I can help you build a product , 
+                          feature or website Look through some of my work and experience! 
+                          If you like what you see and have a project you need coded,  
+                          don’t hestiate to contact me.
+                        </p>
+                  </section>
+                  <section class="inner-section">
+                      <p class="vector4"><img src="image/Vector4.png" alt="horizontal image"></p>
+                      <p class="frame38"><img src="image/Frame38.png" alt="language proficiency"></p>
+                      <p class="vector4"><img src="image/Vector4.png" alt="horizontal image"></p>
+                  </section>
```

Comment:
Kindly note that these skills should not be images instead they should be a list of languages, frameworks, and skills. Kindly fix that and please do not use images for sections of the project that are not supposed to be images 😀

Also kindly fix the layout to have all three lists on the same line in the Desktop version as shown in the Figma design.

Your design:

![image](https://user-images.githubusercontent.com/73714615/187038578-1c6cadbf-13b1-4a43-a7bc-b9e493a77453.png)

Figma design:

![image](https://user-images.githubusercontent.com/73714615/187038590-9b88eded-68d5-443a-a50a-9b0f8e316229.png)